addons/web/static/src/views/list/list_confirmation_dialog.js

```diff
@@ -4,19 +4,17 @@ import { Field, fieldVisualFeedback } from "@web/views/fields/field";
 import { useAutofocus } from "@web/core/utils/hooks";
 
 import { Component } from "@odoo/owl";
+import { FieldOperator } from "@web/core/utils/field_operator";
 
 export class ListConfirmationDialog extends Component {
     static template = "web.ListView.ConfirmationModal";
     static components = { Dialog, Field };
     static props = {
         close: Function,
         title: {
-            validate: (m) => {
-                return (
-                    typeof m === "string" ||
-                    (typeof m === "object" && typeof m.toString === "function")
-                );
-            },
+            validate: (m) =>
```

Comment:
we could take the shot to improve the m2m case as well in the confirmation dialog, which is quite similar to an operation on numeric fields (i.e. adding/removing tags). For now, it displays the resulting tags for the edited record, no the add/remove operation that will be performed on each record. We could re-think the API of this dialog, to make both cases work. Same for dates.